mesh_segmenter/cmake/mesh_segmenter-config.cmake.in

```diff
@@ -0,0 +1,12 @@
+@PACKAGE_INIT@
+
+set(@PROJECT_NAME@_FOUND ON)
+set_and_check(@PROJECT_NAME@_INCLUDE_DIRS "${PACKAGE_PREFIX_DIR}/include")
+set_and_check(@PROJECT_NAME@_LIBRARIES "${PACKAGE_PREFIX_DIR}/lib")
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}")
+
+include(CMakeFindDependencyMacro)
+find_dependency(VTK 7.1)
```

Comment:
I'm not sure you want to explicitly specify the version here. I think this repository builds fine with versions of VTK higher than 7.1 as well